nemo/collections/nlp/data/dialogue/data_processor/data_processor.py

```diff
@@ -50,3 +53,29 @@ def get_dev_examples(self):
     def get_test_examples(self):
         """Gets a collection of `InputExample`s for the test set."""
         raise NotImplementedError()
+
+    @staticmethod
+    def get_relevant_idxs(dataset_split, n_samples, dev_proportion):
+        """
+        Obtain indexes for each dataset_split, when train and dev sets are not in separate files
+        
+        Args: 
+            dataset_split: train, dev or test
+            n_samples: total number of samples
+            dev_proportion: value from 1 to 99 that represent proportion of data in dev set
```

Comment:
Can you please document "Returns:" of this method too?

Reply:
done!